src/ripple/protocol/impl/tokens.cpp

```diff
@@ -241,5 +361,364 @@ decodeBase58Token(std::string const& s, TokenType type)
     // Skip the leading type byte and the trailing checksum.
     return ret.substr(1, ret.size() - 1 - guard.size());
 }
+}  // namespace b58_ref
+
+#ifndef _MSC_VER
+// The algorithms use gcc's int128 (fast MS version will have to wait, in the
+// meantime MS falls back to the slower reference implementation)
+namespace b58_fast {
+namespace detail {
+B58Result<std::span<std::uint8_t>>
+b256_to_b58(std::span<std::uint8_t const> input, std::span<std::uint8_t> out)
```

Comment:
I'd like to see base and endianness info in either the type name (my preference) or variable name, e.g.

```cpp
b256be_to_b58le(b256be const input, b58le out)
```

Reply:
Yes, agreed - although in this function both the input and output are big endian. I changed the name to `b256_to_b58_be` and added a comment. Will push the patch shortly.